jsonnet/kube-prometheus/components/k8s-control-plane.libsonnet

```
@@ -7,6 +7,7 @@ local defaults = {
   commonLabels:: {
     'app.kubernetes.io/name': 'kube-prometheus',
     'app.kubernetes.io/part-of': 'kube-prometheus',
+    'app.kubernetes.io/component': 'control-plane',
```

thinking more about it, some of the resources are about Kubelet and CoreDNS. Maybe this suggestion is more generic (noting that the `app.kubernetes.io/name` labels already tell about the specific component)?
```suggestion
    'app.kubernetes.io/component': 'kubernetes',
```